luminoth/train.py

```diff
@@ -8,6 +8,7 @@
 from tensorflow.python import debug as tf_debug
 
 from luminoth.datasets import get_dataset
+from luminoth.tools.dataset.dataset import InvalidDataDirectory
```

Comment:
I think it's better to import `InvalidDataDirectory` in `luminoth.tools.dataset.__init__.py` and change this to `from luminoth.tools.dataset import InvalidDataDirectory`. If we do this change, we should change the import statement in `luminoth/datasets/tfrecord.py` as well.